conftest.py

```diff
@@ -34,6 +34,18 @@ def __str__(self):
 def pytest_addoption(parser):
     # Create the cfme option group for use in other plugins
     parser.getgroup('cfme', 'cfme: options related to cfme/miq appliances')
+    # Keeping due to compatibility
+    parser.addoption("--no-tracer", dest="no_tracer", action="store_true", default=False,
+                     help="Disable the function tracer (REMOVED, doesn't do anything!)")
+    parser.addoption("--tracer", dest="tracer", action="store_true", default=False,
+                     help="Enable the function tracer (REMOVED, doesn't do anything!)")
+
+
+def pytest_cmdline_main(config):
+    for opt in ['--tracer', '--no-tracer']:
+        if config.getoption(opt) is True:
```

Comment:
@jkrocil in general its considered bad to explicitly check for the boolean value in boolean context,

i think more recent pyflakes/flake8 even warns on it